frontend/public/components/utils/selector-input.jsx

```diff
@@ -23,7 +23,16 @@ export class SelectorInput extends React.Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
I'm unsure of the changes to this component, particularly this one. We don't want to overwrite any edits the user has made.